examples/stateful-counter/README.md

```diff
@@ -0,0 +1,210 @@
+int 1
++
+
+// store to scratch space
+dup
+store 0
+
+// update global state
+app_global_put
+
+// load return value as approval
+load 0
+return
+
+```
+
+This program has a single global key/value pair to store the number of times the program was called. The key is the string "counter" and the value will be defined as an integer when the application is created.
+
+
+# 2. Create a file with Clear Program
+
+Now, we will create clear program. This program does not evaluate any conditions and simply approves the call. Put the code below in `/assets/clear_program.teal`:
+
+```
+#pragma version 2
+
+int 1
+return
+```
+
+# 3. Deploy New Application
+
+To deploy the contract use the following code in a new file named `deploy.js` in `scripts` folder of your project directory:
+
+### Create user accounts
+
+```javascript
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const creatorAccount = deployer.accountsByName.get('alice');
+
+  const algoTxnParams = {
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: creatorAccount.addr,
+    amountMicroAlgos: 200000000,
+    payFlags: {}
+  };
+  // transfer some algos to creator account
+  await executeTransaction(deployer, algoTxnParams);
+```
+
+In the code above we declare user accounts and fund `creatorAccount` account. `masterAccount` is the default account used in algob private net.
+
+### Deploy stateful contract
+
+Firstly we need to put some money into the contract. `master` account will fund it:
+
+```javascript
+  // Create Application
+  // Note: An Account can have maximum of 10 Applications.
+  const sscInfo = await deployer.deploySSC(
+    'approval_program.teal', // approval program
+    'clear_program.teal', // clear program
+    {
+      sender: creatorAccount,
+      localInts: 1,
+      localBytes: 1,
+      globalInts: 1,
+      globalBytes: 1
+    }, {});
+
+  console.log(sscInfo);
+```
+
+Parameters passed are:
+  - Approval Program
+  - Clear Program
+  - SSC(stateful smart contract) arguments which includes sender and details for storage usage.
+  - Transaction parameters
+
+- In above code we have deployed a new application.
+
+- After deploying application you will get the following information on your terminal:
+
+```
+Created new app-id: 189
+{
+  creator: 'EDXG4GGBEHFLNX6A7FGT3F6Z3TQGIU6WVVJNOXGYLVNTLWDOCEJJ35LWJY',
+  txId: 'BCP3ZKT26K2BEB475OMKQHOMBEFVWEHESRS3XMAWQEVASFP6JFUA',
+  confirmedRound: 13899,
+  appID: 189
+}
+
+```
+
+# 4. Opt-In to Application
+
+To Opt-In to an application use the following code in one of your scripts (in `./scripts`):
+
+```javascript
+	await deployer.optInToSSC(Account, applicationID, {}, {});
+```
+
+where `Account` is the account you want to opt-in and applicationID is application index.
+
+# 5. Call Application
+
+To call an application use the following code in one of your scripts (in `./scripts`):
+
+```javascript
+	const tx = {
```

Comment:
Is the indentation here correct? Could you please check?

Reply:
fixed